spring-ai/postman/Spring_AI_Poetry.postman_environment.json

```diff
@@ -0,0 +1,25 @@
+{
+	"id": "df61838b-eb6f-4243-87ee-ca02c77e8646",
+	"name": "Spring_AI_Poetry",
+	"values": [
+		{
+			"key": "baseUrl",
+			"value": "localhost:8080",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "genre",
+			"value": "liric",
+			"enabled": true
+		},
+		{
+			"key": "theme",
+			"value": "flames",
+			"enabled": true
+		}
+	],
+	"_postman_variable_scope": "environment",
+	"_postman_exported_at": "2023-11-26T19:49:21.755Z",
+	"_postman_exported_using": "Postman/10.20.3"
+}
```

Comment:
minor: aren't we missing an empty line after? for lint verification purposes.. not sure though

Reply:
It's generated. Do we really need to update them?

Reply:
its fine, tks